ui/apps/everest/src/pages/database-form/database-preview/database-preview.types.ts

```diff
@@ -27,6 +28,7 @@ export type PreviewSectionProps = {
   children: React.ReactNode;
   active?: boolean;
   hasBeenReached?: boolean;
+  disabled: boolean;
```

Comment:
```suggestion
  disabled?: boolean;
```
to avoid errors during build in unit tests 
![image](https://github.com/percona/everest/assets/90199600/6af06888-e2e3-42ec-85cd-a350387049ee)
ui/apps/everest/src/pages/database-form/database-preview/preview-section.test.tsx